plugin/trino-bigquery/src/main/java/io/trino/plugin/bigquery/BigQueryTypeManager.java

```diff
@@ -380,6 +402,65 @@ private Optional<ColumnMapping> convertToTrinoType(Field field)
         }
     }
 
+    public List<ColumnMetadata> convertToTrinoType(List<String> names, List<String> types)
```

Comment:
Could you add some tests to `TestBigQueryType`? e.g.
```java
    private static final BigQueryTypeManager TYPE_MANAGER = new BigQueryTypeManager(TESTING_TYPE_MANAGER);
...
    @Test
    void testConvertToTrinoType()
    {
        assertColumnType("BOOL", BOOLEAN);
        assertColumnType("INT64", INTEGER);
...
    }

    private static void assertColumnType(String typeString, Type expected)
    {
        ColumnMetadata column = TYPE_MANAGER.convertToTrinoType(List.of("col"), List.of(typeString)).stream().collect(onlyElement());
        assertThat(column.getType()).isEqualTo(expected);
    }
```

Reply:
Done!